src/client/api/event.c

```diff
@@ -406,8 +406,7 @@ daos_event_launch(struct daos_event *ev)
 	if (daos_handle_is_valid(evx->evx_eqh)) {
 		eqx = daos_eq_lookup(evx->evx_eqh);
 		if (eqx == NULL) {
-			D_ERROR("Can't find eq from handle %"PRIu64"\n",
-				evx->evx_eqh.cookie);
+			D_ERROR("Can't find eq from handle %"PRIu64"\n", evx->evx_eqh.cookie);
```

Comment:
minor

s/%"PRIu64"/"DF_U64"

Reply:
i didn't change anything there other than move the argument to the same line due to the increase in width ;-)